Source/UI/VirtusizeView.swift

```diff
@@ -26,65 +26,54 @@ import WebKit
 
 /// A protocol for the Virtusize specific views such as `VirtusizeButton` and `VirtusizeInPageView`
 public protocol VirtusizeView {
-    var style: VirtusizeViewStyle { get }
-    var presentingViewController: UIViewController? { get set }
-    var messageHandler: VirtusizeMessageHandler? { get set }
-	var memoryAddress: String { get }
-	var isDeallocated: Bool? { get set }
-
-	/// Sets up the loading UI
-	func isLoading()
+	var style: VirtusizeViewStyle { get }
+	var presentingViewController: UIViewController? { get set }
+	var messageHandler: VirtusizeMessageHandler? { get set }
+	var clientProduct: VirtusizeProduct? { get set }
```

Comment:
this clientProduct is bound with this VirtusizeView and only gets updated to a different product by the `Virtusize.setVirtusizeView` method used by clients.